internal/compiler/parser/document.rs

```diff
@@ -7,12 +7,12 @@ use super::r#type::parse_struct_declaration;
 
 #[cfg_attr(test, parser_test)]
 /// ```test,Document
-/// Type := Base { }
+/// component Type { }
 /// Type := Base { SubElement { } }
 /// Comp := Base {}  Type := Base {}
-/// Type := Base {} export { Type }
+/// component Q {} Type := Base {} export { Type }
 /// import { Base } from "somewhere"; Type := Base {}
-/// struct Foo := { foo: foo }
+/// struct Foo { foo: foo }
```

Comment:
Is it intentional to remove test coverage for the old syntax?

Reply:
i wanted to have a mix of both, there is still tests with the old syntax when parsing struct specifically.